Controller/CRUDController.php

```diff
@@ -120,12 +120,13 @@ public function listAction()
      * Execute a batch delete.
      *
      * @param ProxyQueryInterface $query
+     * @param Request $request
      *
      * @return RedirectResponse
      *
      * @throws AccessDeniedException If access is not granted
      */
-    public function batchActionDelete(ProxyQueryInterface $query)
+    public function batchActionDelete(ProxyQueryInterface $query, Request $request)
```

Comment:
@greg0ire I'm never sure about this: Would setting it to `null` as default solve the issue?